modules/machine-health-checks-resource.adoc

```diff
@@ -39,9 +40,53 @@ format. For example, `prod-node-us-east-1a`.
 <4> Specify the timeout duration for a node condition. If a condition is met for the duration of the timeout, the Machine will be remediated. Long timeouts can result in long periods of downtime for the workload on the unhealthy Machine.
 <5> Specify the amount of unhealthy machines allowed in the targeted pool of
 machines. This can be set as a percentage or an integer.
+<6> Specify the timeout duration that a MachineHealthCheck must wait for a node to join the cluster before a Machine is determined to be unhealthy.
 
 [NOTE]
 ====
 The `matchLabels` are examples only; you must map your machine groups based on
 your specific needs.
 ====
+
+[id="machine-health-checks-short-circuiting_{context}"]
+== Short-circuiting MachineHealthCheck remediation
```

Comment:
You could remove the hyphen in this title. I.e. "Short circuiting MachineHealthCheck remediation".